_posts/2016-09-20-markdown.md

```diff
@@ -0,0 +1,31 @@
+---
+title: The next chapter for Markdown
+excerpt: Apiary is embracing the CommonMark Markdown syntax for API description.
+layout: post
+date: 2017-09-20 00:00:00 +0200
+author: zdenek
+published: true
+---
+
+
+Apiary has always stood for Markdown. We love the format for its simplicity and accessibility. We’ve built the incredibly popular API description language, API Blueprint on top of it. Markdown is in everything we do. From describing an API, doing a code review or writing this article. Markdown is our DNA.
+
+As the original idea of Markdown evolved over the years we were happy to embrace the GitHub-flavored Markdown syntax to simplify the writing of an API description even more.
+
+Embracing CommonMark
```

Comment:
Should this be a heading @zdne? Currently it's just on it's own so it I think it will be merged with the paragraph below in Markdown and be rendered as a paragraph:

> Embracing CommonMark Today is a milestone in...